examples/elm-pb/data/BOUT.inp

```diff
@@ -50,13 +49,7 @@ upwind = W3
 # Fastest setting depends on machine and MPI
 # implementation. Do not affect result.
 [output]
-ShiftOutput = true
-
-[comms]
-
-async = false           # Use asyncronous sends?
-pre_post = false        # Post receives as early as possible
-group_nonblock = false  # Use non-blocking group operations?
+shiftOutput = true
```

Comment:
Should `shiftOutput` be changed to `shiftoutput` ? It looks like most CamelCase inputs have just become all lower case